tools/restore/restore.sh

```diff
@@ -220,10 +231,37 @@ function restore_database {
     echo "RESTORE OF $db COMPLETE"
 }
 
+function activate_gcp() {
+  echo "Activating google credentials before beginning"
+  gcloud auth activate-service-account --key-file "/credentials/gcp/credentials.json"
+
+  if [ $? -ne 0 ]; then
+    echo "Credentials failed; no way to copy to google."
+    exit 1
+  fi
+}
+
+function activate_aws() {
+  echo "Activating aws credentials before beginning"
+  mkdir -p /root/.aws/
+  cp /credentials/aws/credentials ~/.aws/config
```

Comment:
See above comment. Key file can be in a single stable mount path that's predictable, irrespective of cloud. I.e. `/credentials/secret`.   Relative to these containers, the bit about google's credentials being json is pretty much immaterial